src/setup/serverless-config.go

```diff
@@ -132,13 +137,30 @@ func (s *Serverless) CreateServerlessConfigFile(path string) {
 }
 
 // RemoveService removes the service defined in serverless.yml
-func RemoveService(path string) string {
-	slsRemoveCmd := exec.Command("sls", "remove")
-	slsRemoveCmd.Dir = path
-	slsRemoveMessage := util.RunCommandAndLog(slsRemoveCmd)
-	// cleanup
-	util.RunCommandAndLog(exec.Command("rm", fmt.Sprintf("%sserverless.yml", path)))
-	return slsRemoveMessage
+func RemoveService(provider string, path string) string {
+	switch provider {
+	case "gcr":
+		cmd := fmt.Sprintf("gcloud run services list --region %s | awk '{print $2}' | awk NR\\>1", GCR_DEFAULT_REGION)
+		getServicesCommand := exec.Command("bash", "-c", cmd)
+		lines := util.RunCommandAndLog(getServicesCommand)
+		services := strings.Split(lines, "\n")
+		services = services[:len(services)-1] // Remove last empty line
+		for _, service := range services {
+			log.Infof("Deleting GCR service %s...", service)
+			deleteServiceCommand := exec.Command("gcloud", "run", "services", "delete", "--quiet", "--region", GCR_DEFAULT_REGION, service)
+			deleteMessage := util.RunCommandAndLog(deleteServiceCommand)
+			log.Infof(deleteMessage)
+		}
+		return "All GCR services deleted."
```

Comment:
Do you think we should extract this part as a function? In a future PR `RemoveService` is different for AWS and Azure as well